LibreNMS/Util/IP.php

```diff
@@ -73,6 +73,28 @@ public static function fromHexString($hex, $ignore_errors = false)
         return null;
     }
 
+    /**
+     * Convert a decimal-string to an IP address. For example: "192 168 1 154" -> 192.168.1.254
+     * @param string $snmpOid
+     * @param bool $ignore_errors Do not throw exceptions, instead return null on error.
+     * @return IP|null
+     * @throws InvalidIpException
+     */
+    public static function fromSnmpString($snmpOid, $ignore_errors = false)
```

Comment:
Is this function applicable to more than one code location?  Does it work for both IPv4 and IPv6?

Reply:
https://github.com/librenms/ip-util